is_windy.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+'''
+Program to create dataframe with 'wind' column that is True when rain >0
```

Comment:
Typo at the end of this line; should be "wind > 0 " I believe or maybe "wind > some_limit".